doc/source/install/upgrade_from_5.5.rst

```diff
@@ -0,0 +1,203 @@
+.. _upgrade_from_5.5:
+
+=================================================
+Upgrading Percona XtraDB Cluster from version 5.5
+=================================================
+
+|PXC| versions 5.6 and later have incompatible changes compared to version 5.5.
+If possible, you should stop all nodes, upgrade them, bootstrap the first node,
+and start the others as normal.
+However, to upgrade the cluster from |PXC| version 5.5 without downtime,
+you need to use special configuration options
+that would allow replication between 5.5 nodes and nodes with later versions.
+These compatibility options affect performance,
+so only use them during the upgrade.
+
+This guide describes the procedure for a rolling upgrade (without downtime)
+of a cluster running |PXC| version 5.5 to the latest |PXC| version 5.7.
+The procedure will work similarly for upgrading to 5.6,
+but it is recommended to use the latest available version.
+
+.. note:: Replication between 5.6 and 5.7 is fully compatible.
+   You can have nodes running both versions in the cluster at the same time
+   without any compatibility options required.
+   For information, see :ref:`upgrading_from_5.6`.
+
+To avoid automatic SST while upgrading nodes,
+set the :variable:`wsrep_sst_method` variable to ``skip``.
+You can perform SST manually during the ugrade if necessary:
+
+* If DONOR is 5.5 and JOINER is 5.6/5.7, perform ``mysql_upgrade`` on JOINER
+
+* If JOINER is 5.5 and DONOR is 5.6/5.7, perform full package upgrade on JOINER
+
+However, it is recommended to have large gcache and avoid SST altogether
+until you upgrade all the nodes in the cluster.
+
+To upgrade the cluster from |PXC| version 5.5 without downtime:
+
+.. contents::
+   :local:
+
+Stage 1. Upgrade a single node
+==============================
+
+1. Make sure that all nodes are synchronized.
+
+#. Stop the ``mysql`` process:
+
+   .. prompt:: bash
+
+      sudo service mysql stop
+
+#. Remove version 5.5 packages and install version 5.7.
+   For more information, see :ref:`install`.
+
+   For example, if you are using Percona software repositories,
+   you might use the following commands:
+
+   * On CentOS or RHEL:
+
+     .. code-block:: none
+
+        sudo yum remove 'Percona*'
+        sudo yum install Percona-XtraDB-Cluster-57
+
+   * On Debian or Ubuntu:
+
+     .. code-block:: none
+
+        sudo apt-get remove percona-*
+        sudo apt-get install percona-xtradb-cluster-57
+
+#. Remove or replace variables in the MySQL configuration file :file:`my.cnf`
+   that are not compatible with Percona Server 5.6 and later versions.
+   For more information,
+   see `Changed in Percona Server 5.6 <http://www.percona.com/doc/percona-server/5.6/changed_in_56.html>`_
```

Comment:
a. Should this be 5.7 link
b. So ideally we want a link to PXC-5.7 and given that PXC-5.7 link should list couple of PXC changes and take user to PS-5.7 link for PS or MySQL changes ?